src/Module/AccMgmtControlDoc.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+
+/**
+ * Static definition for the Firefox Account Manager
+ *
+ * @see https://wiki.mozilla.org/Labs/Weave/Identity/Account_Manager/Spec/3#Contents_of_the_Account_Management_Control_Document
+ */
+class AccMgmtControlDoc extends BaseModule
```

Comment:
Please rename to the full name, e.g. `AccountManagementControlDocument`. We aren't paying more for longer names!

Reply:
This will become a real long name. I though it would be better listed in the Module directory if I shorten it a little bit.

But I'll change it :)

Reply:
I had to squint really hard to understand this heavily shortened name, I'm not good without vowels.